config/crd/bases/batch.volcano.sh_jobs.yaml

```diff
@@ -6857,6 +6862,9 @@ spec:
                           - containers
                           type: object
                       type: object
+                    topologyPolicy:
```

Comment:
suggest to list all available policy in the describtion

Reply:
these files are auto generated by scripts

Reply:
the topologyPolicy is added by me in api repo , but the PR hasn't been merged in volcano

Reply:
emmm yeah, but it has been merged into volcano/apis, so the script generated these definitions..